web_app/dist/main.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="icon" type="images/png" href="images/icon.png" />       
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TaskViewer</title>
+        <meta name="description" content="Schedule tasks, without the clutter">
+        <link rel="stylesheet" href="main.css">
+        
+    </head>
+ 
+    <body>  
+        
+    <div id="sticky-header">
+        <div class="navbar">
+            <img id="logo" src="images/icon_small.png" alt="TaskViewer Logo">
+            <a>Log out</a>
+            <a>Sort</a>
+            <a>Filter by</a>
+            <input id="search-bar" type="text" placeholder="Search..">
+        </div>
+        <div class="new-flex-row-anchor-right">
```

Comment:
Just a thought, we might be able to combine this and the other `div` into one:

```html
<div class="button-anchor-right-container">
    <button>...</button>
    <button>...</button>
</div>
```

Of course, CSS would need to change to make this work.

Reply:
the "other div"? huh?

Reply:
There's two divs with the new flex row class, I'm saying we might be able to make it just one instead using some CSS